src/Container/ServiceProvider.php

```diff
@@ -63,7 +63,6 @@ public function __invoke(ContainerInterface $container): void
         $container->tag(NameResolver::class, [NodeVisitor::class]);
         $container->tag(CloningVisitor::class, [NodeVisitor::class]);
         $container->tag(ParentConnectingVisitor::class, [NodeVisitor::class]);
```

Comment:
same line already here

Reply:
Nice catch @staabm 

Thanks for your help.